po/fr.po

```diff
@@ -184,12 +184,12 @@ msgid "Failed to mount again the disk. Aborting the format operation."
 msgstr "Impossible de remonter le disque. Abandon de l’opération de formatage."
 
 #: src/choose_disk.vala:117
-#, fuzzy
 msgid "Failed to get the final UUID. Aborting the format operation."
-msgstr "Impossible de remonter le disque. Abandon de l’opération de formatage."
+msgstr ""
+"Impossible de trouver l'UUID final. Abandon de l’opération de formatage."
```

Comment:
There is one more `\n` that you added

Reply:
stranger. That was my translating software. Corrected in next commit.